eldritch/places.py

```diff
@@ -91,8 +91,8 @@ def CreatePlaces():
   Shop = Location("Shop", "Curiositie Shoppe", None)
   Northside = Street("Northside", "Northside")
   Northside._add_connections(Newspaper, Train, Shop)
-  Bank = Location("Bank", "Bank", None)
-  Asylum = Location("Asylum", "Asylum", None)
+  Bank = Location("Bank", "Bank of Arkham", None)
```

Comment:
The word Arkham should not appear in the source code.

Reply:
removed